### DISCUSSION TITLE
Two sided student ID

### DISCUSSION THREAD

**Discussion by koma52** [community/community#77987] [GitHub Education]
### Hi everyone,
My student ID card is two sided and my name and the dates are on the opposite sides. I can only upload one picture on the form but if I try to edit the two sides on one picture it detects that the image was edited. What should I do?

**Comment by Hugoalv79** [comment]
Call support assistance bro, it's something too much specific.

**Reply by koma52** [reply]
Where can I do that? I didn't find any contact information for education support just this forum and the support bot which is kind of useless

**Comment by mahfuz0001** [comment]
You can contact Github support at [Github Education Support](https://support.github.com/features/education?tags=dotcom-footer)
